src/feature/footer/footer.tsx

```diff
@@ -0,0 +1,58 @@
+import { DribbbleIcon, GithubIcon, TwitterIcon } from 'lucide-react';
+import { Link } from 'react-router';
+
+function Footer() {
+  return (
+    <div className="flex min-h-screen flex-col">
+      <div className="bg-muted grow" />
+      <footer>
+        <div className="mx-auto max-w-screen-xl">
+          <div className="flex flex-col-reverse items-center justify-between gap-x-2 gap-y-5 px-6 py-8 sm:flex-row xl:px-0">
+            <span className="text-muted-foreground">
+              &copy; {new Date().getFullYear()}{' '}
+              <Link to="/" className="hover:text-chart-3 transition-colors duration-300">
+                Poster Store
+              </Link>
+              . All rights reserved.
```

Comment:
мне кажется тут точка в конце лишняя
```suggestion
              . All rights reserved
```

Reply:
поправил